app/assets/javascripts/geoblacklight/downloaders/downloader.js

```diff
@@ -34,12 +37,14 @@
     complete: function(data) {
       this.downloading = false;
       this.$el.prop('disabled', false);
+      this.$el.prop('outerHTML', data[0][1]);
```

Comment:
this replaces the `Preparing download` message with the actual link to the download itself when it is ready ... it is in the response that is also used to generate the flash alert message 

response from server looks something like this, coming from here: https://github.com/geoblacklight/geoblacklight/blob/main/app/controllers/download_controller.rb#L27-L30

```
[
    [
        "success",
        "\u003ca data-download=\"trigger\" data-download-id=\"stanford-cz128vq0535\" data-download-type=\"generated-geojson\" href=\"/download/file/stanford-cz128vq0535-geojson.json\"\u003eYour file stanford-cz128vq0535-geojson.json is ready for download\u003c/a\u003e"
    ]
]
```

Reply:
Could also just make this a simple message like, and the user would have to get the actual link from the alert (as they do now).  OR, we could create a new link with any text we want (e.g. link the text "Download ready" to the file itself) though doing this would require adding some extra data to the response from the server to include just the link itself in addition to the full flash message.

```
  this.$el.prop('innerHTML', 'Download ready');
```